src/meshapi/pelias.py

```diff
@@ -4,10 +4,11 @@
 
 import inflect
 import requests
+import environment
 
 from meshapi.util.constants import DEFAULT_EXTERNAL_API_TIMEOUT_SECONDS
 
-PELIAS_ADDRESS_PARSER_URL = os.environ.get("PELIAS_ADDRESS_PARSER_URL", "http://localhost:6800/parser/parse")
+PELIAS_ADDRESS_PARSER_URL = environm
```

Comment:
You should be able to do something like this:
```suggestion
from environment import PELIAS_ADDRESS_PARSER_URL
```